packages/block-library/src/search/index.php

```diff
@@ -42,16 +42,38 @@ function render_block_core_search( $attributes ) {
 	// Border color classes need to be applied to the elements that have a border color.
 	$border_color_classes = get_border_color_classes_for_block_core_search( $attributes );
 
+	$wp_kses_formatting_tags = array(
+		'code'   => array(),
+		'em'     => array(),
+		'img'    => array(
+			'scale' => array(),
+			'class' => array(),
+			'style' => array(),
+			'src'   => array(),
+			'alt'   => array(),
+		),
+		's'      => array(),
+		'span'   => array(
+			'style' => array(),
+		),
+		'strong' => array(),
+	);
+
```

Comment:
```suggestion
	$wp_kses_formatting_tags = array_map( '_wp_add_global_attributes', $wp_kses_formatting_tags ); 

```

Will allow advanced users to add classes, IDs and other pretty safe attributes. See

https://github.com/WordPress/wordpress-develop/blob/f745cc551bb0a54d7df72ec97d23e3359c56048d/src/wp-includes/kses.php#L2559-L2571

Reply:
I've switched it over to `wp_kses_post` now, and it looks like it implicitly has the global attributes:
https://github.com/WordPress/wordpress-develop/blob/f745cc551bb0a54d7df72ec97d23e3359c56048d/src/wp-includes/kses.php#L736

Thanks for the advice!